.circleci/config.yml

```diff
@@ -120,7 +120,7 @@ build_on_tag_or_prerelease: &build_on_tag_or_prerelease
     tags:
       only: /^v.*/
     branches:
-      only: prerelease
+      only: e2e-tests
```

Comment:
Intentionally left until the PR is merged so that tests can be run in both local Kind and GKE.

Reply:
That branch name will not be merged, of course. I left it intentionally while this PR is opened in case we need to push new stuff, that we can run the tests in GKE too.